applications/crossbar/src/modules_v2/cb_phone_numbers_v2.erl

```diff
@@ -597,18 +597,30 @@ maybe_find_numbers(Context) ->
 -spec find_numbers(cb_context:context(), ne_binary(), ne_binary()) -> cb_context:context().
 find_numbers(Context, AccountId, ResellerId) ->
     QS = cb_context:query_string(Context),
+    Country = kz_json:get_ne_value(?COUNTRY, QS, ?KNM_DEFAULT_COUNTRY),
+    Prefix = kz_util:remove_white_spaces(kz_json:get_ne_value(?PREFIX, QS)),
+    Offset = kz_json:get_integer_value(?OFFSET, QS, 0),
+    Token = cb_context:auth_token(Context),
+    HashKey = <<AccountId/binary, "-", Token/binary>>,
+    Hash = kz_base64url:encode(crypto:hash(sha, HashKey)),
+    QueryId = list_to_binary([Country, "-", Prefix, "-", Hash]),
+    Dialcode = knm_util:prefix_for_country(Country),
+    NormalizedPrefix = <<Dialcode/binary, Prefix/binary>>,
```

Comment:
i believe it does if its not filled. anyway, this is the only entry point to knm_search for now